.github/workflows/create_containers.yml

```
@@ -29,36 +58,90 @@ jobs:
+            echo "exists=false" >> "$GITHUB_OUTPUT"
+            echo "Container ${TAG} does not exist."
+          fi
+      - name: Let the user know
+        if: steps.check_container.outputs.exists != 'true' && github.event_name == 'pull_request'
+        run: |
+          TAG="${{steps.tag.outputs.tag}}"
+          echo "::error::The requested image ${TAG} does not yet exist. Wait until the push pipeline is complete, then re-run this job."
+          exit 1
+      # Steps below here are skipped for pull requests because we don't have permission anyway
       - name: Generate Dockerfile
+        if: ${{ github.event_name == 'workflow_call' && inputs.force_rebuild }} || (steps.check_container.outputs.exists != 'true' && github.event_name != 'pull_request')
```

actionlint (1.7.12)

[error] if: condition "${{ github.event_name == 'workflow_call' && inputs.force_rebuild }} || (steps.check_container.outputs.exists != 'true' && github.event_name != 'pull_request')" is always evaluated to true because extra characters are around ${{ }}

(if-cond)